opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/__init__.py

```diff
@@ -66,6 +68,12 @@ def run() -> None:
                 )
                 argument_otel_environment_variable[argument] = attribute
 
+    parser.add_argument(
```

Comment:
When executing command "opentelemetry-instrument --version python app.py" where app.py is a sample app, it will just show the version and ignore the remaining command.
I guess this is not expected behaviour right?

Reply:
I wouldn't expect anybody to use the command like that but still Why do you think it's not expected behaviour?

For example you could run `python --version prog.py` and it just prints version.

Reply:
You are overthinking this small change. Just assume they use this flag like they generally do with any other cli-tool.

Reply:
ok sure